Intents/None.cs

```diff
@@ -19,8 +20,13 @@ internal class None : IntentHandler
             "I don't really know how to respond to that. 😅"
         };
 
-        public override Task Handle(InteractionContext context) =>
-            context.Interaction.RespondAsync(_responses.PickRandom());
-
+        public override Task Handle(InteractionContext context)
+        {  
+            string response = new AIHandler().responseHandler(context);

```

Comment:
Can probably make a new pipeline object for this stuff rather than putting it with the no response stuff. Have a look at the constructor for the DiscordHandler, you can see the list of classes it goes through in the pipeline there.